manubot/process/header-includes-template.html

```diff
@@ -0,0 +1,77 @@
+<!--
+Manubot generated metadata rendered from header-includes-template.html.
+Suggest improvements at https://github.com/manubot/manubot/blob/master/manubot/process/header-includes-template.html
+-->
+<meta name="dc.format" content="text/html">
+{% if title is defined %}
+<meta name="dc.title" content="{{ title|escape }}">
+<meta name="citation_title" content="{{ title|escape }}">
+<meta property="og:title" content="{{ title|escape }}">
+<meta property="twitter:title" content="{{ title|escape }}">
+{% endif %}
+{% if iso_date is defined %}
+<meta name="dc.date" content="{{ iso_date }}">
+<meta name="citation_publication_date" content="{{ iso_date }}">
+{% endif %}
+{% if lang is defined %}
+<meta name="dc.language" content="{{ lang }}">
+<meta name="citation_language" content="{{ lang }}">
+{% endif %}
+<meta name="dc.relation.ispartof" content="Manubot Preprint">
+<meta name="dc.publisher" content="Manubot">
+<meta name="citation_journal_title" content="Manubot Preprint">
+<meta name="citation_technical_report_institution" content="Manubot">
+<!-- author information --> 
+{% for author in authors|default([]) if author.name is defined %}
+<meta name="citation_author" content="{{ author.name|escape }}">
+{% if author.affiliations is defined %}
+{% for affiliation in author.affiliations %}
+<meta name="citation_author_institution" content="{{ affiliation|escape }}">
+{% endfor %}
+{% endif %}
+{% if author.orcid is defined %}
+<meta name="citation_author_orcid" content="{{ author.orcid }}">
+{% endif %}
+{% if author.twitter is defined %}
+<meta name="twitter:creator" content="@{{ author.twitter }}">
+{% endif %}
+{% endfor %}
+{% if html_url is defined %}
+<link rel="canonical" href="{{ html_url }}">
+<meta property="og:url" content="{{ html_url }}">
+<meta property="twitter:url" content="{{ html_url }}">
+<meta name="citation_fulltext_html_url" content="{{ html_url }}">
+{% endif %}
+{% if pdf_url is defined %}
+<meta name="citation_pdf_url" content="{{ pdf_url }}" />
+<link rel="alternate" type="application/pdf" href="{{ pdf_url }}">
+{% endif %}
+{% if html_url_versioned is defined %}
+<link rel="alternate" type="text/html" href="{{ html_url_versioned }}">
+<meta name="manubot_versioned_html_url" content="{{ html_url_versioned }}">
+{% endif %}
+{% if pdf_url_versioned is defined %}
+<meta name="manubot_versioned_pdf_url" content="{{ pdf_url_versioned }}">
+{% endif %}
+{% if description is defined %}
```

Comment:
fallback to abstract

Reply:
done in https://github.com/manubot/manubot/pull/138/commits/77dd81a70a46bd268778ad9c050dff7188cb29a9